### PR TITLE
docs: prepare 3.3 changelog

### PR DESCRIPTION
**Pull request by lengau**
I figured I'd get an early start on this so it's not a last-minute thing.

### CODE REVIEW

File: common.mk [hunk 1]
```diff
@@ -190,11 +190,11 @@ test-coverage:  ## Generate coverage report
 
 .PHONY: docs
 docs:  ## Build documentation
-	uv run --extra docs sphinx-build -b html -W $(DOCS) $(DOCS)/_build
+	uv run --extra docs sphinx-build -b html $(DOCS) $(DOCS)/_build
 
 .PHONY: docs-auto
 docs-auto:  ## Build and host docs with sphinx-autobuild
-	uv run --extra docs sphinx-autobuild -b html --open-browser --port=8080 --watch $(PROJECT) -W $(DOCS) $(DOCS)/_build
+	uv run --extra docs sphinx-autobuild -b html --open-browser --port=8080 --ignore *.kate-swp --ignore docs/reference/commands/** --watch $(PROJECT) $(DOCS) $(DOCS)/_build
```
Comment (medubelko) [inline common.mk:197]: I'm wondering if we could pipe the exclusion list from `docs/conf.py` into the arguments here.

File: docs/reference/changelog.rst [hunk 2]
```diff
@@ -77,14 +77,51 @@ Changelog
...
+
+- The ``charm`` command is now run in verbose mode by default.
+
 Command line
 ============
 
-The pack command now updates charm the libs in the project directory if they don't meet
-the requirements in the ``charm-libs`` key of ``charmcraft.yaml``.
+- The ``pack`` command now updates charm the libs in the project directory if they
+  don't meet the requirements in the ``charm-libs`` key of ``charmcraft.yaml``.
```
Comment (medubelko) [inline docs/reference/changelog.rst:108]:
```suggestion
- The ``pack`` command now updates the charm libaries in the project directory
  if they don't meet the requirements in the ``charm-libs`` key of
  ``charmcraft.yaml``.
```

"charm the libs" sounds like a new slogan...

File: docs/reference/changelog.rst [hunk 3]
```diff
@@ -77,14 +77,51 @@ Changelog
...
 ------------------
 
+Core
+====
+
+Plugins
+#######
+
+- New :ref:`craft_parts_poetry_plugin`
+- New :ref:`craft_parts_python_plugin`
```
Comment (medubelko) [inline docs/reference/changelog.rst:90]:
```suggestion
- New :ref:`craft_parts_poetry_plugin`.
- New :ref:`craft_parts_python_plugin`.
```

**Review by medubelko — CHANGES_REQUESTED** [review]
The suggestion with punctuation is for consistency, but feel free to ignore it.

File: common.mk (hunk 1 above)
Comment (lengau) [reply common.mk:197]: Not without extra work that's out of scope here.

Ideally we'd get [this fixed upstream](https://github.com/sphinx-doc/sphinx-autobuild/issues/140). Likewise, it'd be nice to get [this fixed too](https://github.com/sphinx-contrib/sphinx-lint/issues/66).

Comment (medubelko) [reply common.mk:197]: As I should have guessed, you two have already flagged this. :)

**Review by medubelko — APPROVED** [review]

**Review by bepri — APPROVED** [review]

**Review by mr-cal — APPROVED** [review]

**Pull request merged**